src/components/menu/examples/menu-basic-secondary-text.tsx

```diff
@@ -0,0 +1,66 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * With `secondaryText`
+ *
+ * Menu items can display secondary text as well. By default, the secondary text
+ * will be displayed in two lines, and then get truncated.
+ *
+ * :::important
+ * Keep in mind that menu lists stretch as much as they can to display
+ * their content. You must therefore specify a maximum with for the menus with
+ * long secondary text, otherwise they will fill out the entire screen width.
+ *
+ * But do not forget that menus should behave responsively, thus using a fixed `width`
+ * should be avoided. To make the width responsive, try using the `min()` function.
+ * This function selects the smallest value from a list of comma-separated expressions
+ * which are placed within the parentheses.
+ *
+ * For example, `--menu-surface-width: min(90vw, 40rem);` will output
+ * `width: min(90wv, 40rem);` which will tell the browser to render the menu
+ * content in a 40rem-wide grid, as long as 90% of the viewport's width (`90vw`)
+ * is larger than `40rem`.
+ * :::
+ */
+@Component({
+    tag: 'limel-example-menu-secondary-text',
+    shadow: true,
+    styleUrl: 'menu-basic-secondary-text.scss',
+})
+export class MenuBasicExample {
+    @State()
+    private lastSelectedItem: string;
+
+    private items: Array<MenuItem | ListSeparator> = [
+        {
+            text: 'This item only has one line of primary text',
+        },
+        { separator: true },
+        {
+            text: 'Very long primary texts like this one can truncate based on what you specify for `--menu-surface-width`.',
+            secondaryText: 'This is a short secondary text.',
+        },
+        {
+            text: 'This item only has one line of primary text',
+            secondaryText:
+                'The length of secondary text exceeds maximum allowed number of lines, which is two. This happens because `--menu-surface-width` specified here is not so large. Thus the lines will not truncate.',
```

Comment:
I'm not sure what you mean by "Thus the lines will not truncate" here? It looks like this for me:

<img width="374" alt="image" src="https://user-images.githubusercontent.com/154725/151006463-732dfd85-65ad-445b-9aba-fca26082ea2c.png">

Reply:
good catch. ~not~ ⚡️